tree/ntupleutil/v7/src/RNTupleInspector.cxx

```diff
@@ -21,7 +21,6 @@
 
 #include <TFile.h>
 
-#include <algorithm>
```

Comment:
I think we should still keep that include because we use it here in the implementation.